src/Xamarin.Android.Build.Tasks/Tasks/ResolveAssemblies.cs

```diff
@@ -95,6 +96,7 @@ bool Execute (DirectoryAssemblyResolver resolver)
 
 			ResolvedAssemblies = assemblies.Select (a => new TaskItem (a)).ToArray ();
 			ResolvedSymbols = assemblies.Select (a => a + ".mdb").Where (a => File.Exists (a)).Select (a => new TaskItem (a)).ToArray ();
+			ResolvedSymbols.Concat (assemblies.Select (a => Path.ChangeExtension (a, "pdb")).Where (a => File.Exists (a) && Files.IsPortablePdb(a)).Select (a => new TaskItem (a)));
```

Comment:
I think this should be:

    ResolvedSymbols = ResolvedSymbols.Concat (...).ToArray();

As-is, it's discarding the returned `IEnumerable<TaskItem>`.

Additionally, please make this a multi-line affair:

    ResolvedSymbols = ResolvedSymbols.Concat (
            assemblies.Select (a => Path.ChangeExtension (a, "pdb"))
            .Where (a => File.Exists (a) && Files.IsPortablePdb(a))
            .Select (a => new TaskItem (a)))
        .ToArray ();